python/cudf/cudf/tests/test_orc.py

```diff
@@ -670,3 +673,29 @@ def test_orc_reader_gmt_timestamps(datadir):
     pdf = orcfile.read().to_pandas()
     gdf = cudf.read_orc(path, engine="cudf").to_pandas()
     assert_eq(pdf, gdf)
+
+
+def test_orc_bool_encode_fail():
+    np.random.seed(0)
+
+    # Generate a boolean column longer than a single stripe
+    fail_df = cudf.DataFrame({"col": gen_rand_series("bool", 600000)})
+    # Invalidate the first row in the second stripe to break encoding
+    fail_df["col"][500000] = None
```

Comment:
Is there a way to programmatically get this number to be the first row in the second stripe? What if a default changes in our stripe sizing in the future?

Reply:
There's no way to get this grammatically ATM. I think exposing the stripe size as parameter would make this possible, but it would probably still not be very clean to derive the row count.
I'm hoping that this test will soon be removed as the underlying issue is fixed.